recipes/evalica/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "evalica" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/evalica-{{ version }}.tar.gz
+  sha256: 49ce78da9dea0b75ece86ba0ba2159ee3550a2c604b73675db82a064e1f68164
+
+build:
+  entry_points:
+    - evalica = evalica.__main__:main
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('rust') }}
+    - python  # [build_platform != target_platform]
+    - cargo-bundle-licenses
+    - maturin
+    - numpy
+    - pandas
+  host:
+    - python
+    - pip
+    - maturin
+  run:
+    - python
+    - numpy
+    - pandas
+    - libgcc-ng  # [linux]
```

Comment:
Please add a c compiler instead.
```suggestion
```

Reply:
I followed the example of other recipes that approached this problem this way: https://github.com/search?q=org%3Aconda-forge+libgcc-ng+%5Blinux%5D&type=code.

I tried adding the compiler to `run` as `{{ compiler('c') }}`, but did not seem to work:

```
conda_build.exceptions.OverLinkingError: overlinking check failed 
["  ERROR (evalica,lib/python3.10/site-packages/evalica/evalica.cpython-310-x86_64-linux-gnu.so): .. but ['conda-forge/linux-64::libgcc-ng==14.1.0=h77fa898_0'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]
```